packages/create-cloudflare/vitest.d.ts

```diff
@@ -1,12 +1,14 @@
+// See https://vitest.dev/guide/extending-matchers.html
+/* eslint-disable unused-imports/no-unused-imports */
+/* eslint-disable no-shadow */
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import type { Assertion, AsymmetricMatchersContaining } from 'vitest'
```

Comment:
I see that this is what is in the example. But I don't understand why these imports are needed here...
Can you explain?

Reply:
I've changed it to `import vitest`. I'm not fully certain, but I'm pretty sure it needs imported to ensure the `declare module 'vitest'` is merged with existing definitions